From 703aec16c07cf6edf5fe2c637a9faaeb74f96cf6 Mon Sep 17 00:00:00 2001
From: Valentin Ciocoi Radulescu <valentin.ciocoiradulescu@nxp.com>
Date: Mon, 29 Jun 2026 17:19:33 +0300
Subject: [PATCH] s32g399ardb3: append bootargs for pcie-services-demos

Append 'maxcpus=6 pci=pcie_bus_safe' to the kernel command line via the
EXTRA_BOOT_ARGS hook, which s32.h concatenates onto mmcargs (and the
other bootargs variants). Enabled for the pcie-services-demos build
only, wired through the Yocto u-boot recipe.

Signed-off-by: Valentin Ciocoi Radulescu <valentin.ciocoiradulescu@nxp.com>
---
 include/configs/s32g399ardb3.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32g399ardb3.h b/include/configs/s32g399ardb3.h
index 42c8ed8ba8e..d563d7ecae8 100644
--- a/include/configs/s32g399ardb3.h
+++ b/include/configs/s32g399ardb3.h
@@ -8,7 +8,7 @@
 
 #include <configs/s32g3.h>
 
-#define EXTRA_BOOT_ARGS			""
+#define EXTRA_BOOT_ARGS			" maxcpus=6 pci=pcie_bus_safe"
 
 #ifndef CONFIG_NXP_PFENG_SLAVE
 #define FDT_FILE			CONFIG_DTB_FILE
-- 
2.25.1

